doc/using/configuration.chapter.md

```diff
@@ -264,8 +264,9 @@ This provides us with some useful documentation for using our packages.  However
 {
   packageOverrides = pkgs: with pkgs; rec {
     myProfile = writeText "my-profile" ''
-      export PATH=$HOME/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/sbin:/bin:/usr/sbin:/usr/bin
-      export MANPATH=$HOME/.nix-profile/share/man:/nix/var/nix/profiles/default/share/man:/usr/share/man
+      if [ -z "$XDG_DATA_HOME" ]; then XDG_DATA_HOME=$HOME/.local/share; fi
```

Comment:
bash also supports the following syntax

``${XDG_DATA_HOME:-$HOME/.local/share}``

Reply:
True, totally forgot about that!